lib/musicalbum.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'item'
+
+class MusicAlbum < Item
+  attr_accessor :on_spotify
+
+  def initialize(published_date, id = SecureRandom.uuid, on_spotify: true, archived: false)
+    super(published_date, id, archived: archived)
+    @on_spotify = on_spotify
+  end
+
+  def can_be_archived?
+    true if @on_spotify
```

Comment:
Optional: you could return @on_spotify and it will still work.

Reply:
Thanks i will do that